spec/02-integration/05-proxy/10-balancer/03-consistent-hashing_spec.lua

```diff
@@ -153,4 +153,4 @@ for _, strategy in helpers.each_strategy() do
       end)
     end)
   end
-end
\ No newline at end of file
+end
```

Comment:
 🤔how didn't this break the build before?

Reply:
You may be misreading the diff: this is just adding a line break after the `end`.